dbos/_workflow_commands.py

```diff
@@ -118,3 +123,31 @@ def fork_workflow(
         application_version=application_version,
     )
     return forked_workflow_id
+
+
+def garbage_collect(
+    dbos: "DBOS",
+    time_threshold_ms: Optional[int],
+    rows_threshold: Optional[int],
+) -> None:
+    if time_threshold_ms is None and rows_threshold is None:
+        return
+    result = dbos._sys_db.garbage_collect(
+        time_threshold_ms=time_threshold_ms, rows_threshold=rows_threshold
+    )
+    if result is not None:
+        cutoff_epoch_timestamp_ms, pending_workflow_ids = result
+        dbos._app_db.garbage_collect(cutoff_epoch_timestamp_ms, pending_workflow_ids)
+
+
+def global_timeout(dbos: "DBOS", timeout_ms: int) -> None:
```

Comment:
Do we want to expose this in admin server as well? Then we can automatically cancel from cloud/conductor.

Reply:
Done!